### PR TITLE
Jonasthiel patch 1

### PR DESCRIPTION
**Pull request by ghost**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [x] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by jonasthiel** [comment]
This cask 'clipmenu.rb' is a return of the beloved app ClipMenu.

File: Casks/clipmenu.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'clipmenu' do
+  version '0.4.3'
+  sha256 '4f26de434a8d412bb063515b6f9354ae4eed12c265af36eefb83e2bfe3647eac'
+
+  # dropbox.com/u/1140644 was verified as official when first introduced to the cask
+  url "https://www.dropbox.com/s/mz2v2psimbwdfql/ClipMenu.app.zip?dl=1"
```
Comment (commitay) [inline Casks/clipmenu.rb]: Where did you get this URL?

Comment (jonasthiel) [reply Casks/clipmenu.rb]: Good question, I have found it online -- right now it is on a private dropbox.

Comment (commitay) [reply Casks/clipmenu.rb]: To re-add this Cask the download URL would need to be official.

Comment (jonasthiel) [reply Casks/clipmenu.rb]: Ok, thanks. I'll try to contact the developer.

**Pull request closed without merging**